static/app/views/settings/account/notifications/fields2.tsx

```diff
@@ -177,4 +178,18 @@ export const QUOTA_FIELDS = [
       ['never', t('Off')],
     ] as const,
   },
+  {
+    name: 'quotaSpendAllocations',
+    label: (
+      <Fragment>
+        {t('Spend Allocations')}{' '}
+        <QuestionTooltip position="top" title="Business plan only" size="xs" />
```

Comment:
Do you have a screenshot of what this looks like? Also a badge might be better since it doesn't require user interaction to mention it's only for business plan.

Reply:
added a screenshot!

Reply:
the feature badges we have are "new", "alpha", and "beta", and you can only customize the tooltip anyway 😢